cmd/server-main.go

```diff
@@ -61,6 +61,9 @@ ENVIRONMENT VARIABLES:
   REGION:
      MINIO_REGION: To set custom region. By default it is "us-east-1".
 
+  ENDPOINT:
+     MINIO_ENDPOINT: To set location endpoint of the instance.
```

Comment:
I think `MINIO_HOST_NAME` would be nice.  However we will discuss and take it forward

Reply:
The reason i didn't want it as HOST_NAME is because ENDPOINT encourages providing PORT as well @balamurugana